### ISSUE TITLE
Cannot use

### ISSUE THREAD

**Issue by TrustedFloppa** [rytilahti/python-miio#1409]
I cannot do anything because of error

 - OS: Windows
 - python-miio: [Name: python-miio
Version: 0.5.11
Summary: Python library for interfacing with Xiaomi smart appliances
Home-page: https://github.com/rytilahti/python-miio
Author: Teemu R
Author-email: tpr@iki.fi
License: GPL-3.0-only
Location: c:\users\acera\appdata\local\programs\python\python310\lib\site-packages
Requires: android_backup, appdirs, attrs, click, construct, croniter, cryptography, defusedxml, netifaces, pytz, PyYAML, tqdm, zeroconf
Required-by:]

**Expected behavior**
A something else appear

**Console output**
C:\Users\acera\OneDrive\Desktop>miiocli --help
Traceback (most recent call last):
  File "C:\Users\acera\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\acera\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\acera\AppData\Local\Programs\Python\Python310\Scripts\miiocli.exe\__main__.py", line 4, in <module>
  File "C:\Users\acera\AppData\Local\Programs\Python\Python310\lib\site-packages\miio\cli.py", line 43, in <module>
    cli.add_command(device_class.get_device_group())
  File "C:\Users\acera\AppData\Local\Programs\Python\Python310\lib\site-packages\miio\integrations\vacuum\roborock\vacuum.py", line 932, in get_device_group
    @dg.resultcallback()
AttributeError: 'DeviceGroup' object has no attribute 'resultcallback'. Did you mean: 'result_callback'?

**Comment by rytilahti** [comment]
Duplicate of #1389. You can either downgrade click, or use the code from master, until a new release is made.
